odoo/addons/base/transpiler/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .js_transpiler import transpile_javascript, is_odoo_module
+from .sourcemap_generator import SourceMapGenerator
```

Comment:
The assets classes kind of make sense in module `base`, in particular because of their interaction with attachments.
However, the transpiler stuff does not depend in any way from `base`.
Wouldn't it make sense to move `js_transpiler` and `sourcemap_generator` into `odoo.tools`?

Reply:
I also suggest to rename `sourcemap_generator` into `js_sourcemap`.